configure.ac

```diff
@@ -2333,16 +2333,26 @@ struct sockaddr_storage {
 #error dont know what to use for SIZE_T_FMT
 #endif
 
-#if (SIZEOF_OFF_T == SIZEOF_LONG)
-#define OFF_T_FMT "%ld"
-#define strtoofft(nptr, endptr, base) strtol(nptr, endptr, base)
-#elif (SIZEOF_OFF_T == SIZEOF_LONG_LONG_INT)
+#if (SIZEOF_OFF_T == SIZEOF_LONG_LONG_INT)
 #define OFF_T_FMT "%lld"
 #define strtoofft(nptr, endptr, base) strtoll(nptr, endptr, base)
+#elif (SIZEOF_OFF_T == SIZEOF_LONG)
+#define OFF_T_FMT "%ld"
+#define strtoofft(nptr, endptr, base) strtol(nptr, endptr, base)
 #else
 #error dont know what to use for OFF_T_FMT
 #endif
 
+#if (SIZEOF_TIME_T == SIZEOF_LONG_LONG_INT)
+#define TIME_T_FMT "%lld"
+#define strtoofft(nptr, endptr, base) strtoll(nptr, endptr, base)
```

Comment:
should not be defined here `time_t` case , defined above `off_t` case.

Reply:
or define `strtotimet` and change to use it for scanning `time_t`, for example, `parse_zoneinfo`
 https://github.com/cyrusimap/cyrus-imapd/blob/95cc49f06b9794012ddba8879f1a842374281b9a/imap/zoneinfo_db.c#L140
(already fixed for `zoneinfo_store` case)

Reply:
@obache thanks! Fixed by removing the duplicate lines (I've altered the 1st commit and force-pushed it).

Reply:
How about `strtol` usage?

Reply:
@elliefm, what about the `strtoofft` function define that got removed with your 2 commits? Was it unused? And what about the `strtol` that @obache mentions?

Reply:
Oh, I missed that hiding in there!  But searching the source tree, looks like it's never used.  Checking history, it was added in 2008, only used in one place, and then stopped being used in 2010.  So it's just cruft, and doesn't hurt to remove it.

I'm not sure about that `strtol` yet, will think about it some more...